packages/components/pages/f-contact-preferences/stories/story.helper.js

```diff
@@ -10,82 +10,86 @@ const httpVerbs = {
     post: 'POST'
 };
 
-export const apiStates = {
-    none: null,
-    apiPostFailed: 'api-post-failed',
-    apiGetFailed: 'api-get-failed'
+const apiStates = {
+    none: 'no-issues',
+    apiGetFailed: 'get-preferences-fails',
+    apiPostFailed: 'post-preferences-fails'
 };
 
 export const apiStateOptions = {
     title: 'Set Api State',
-    states: {
-        None: apiStates.none,
-        'The POST/SUBMIT fails (change a value then click `Save Changes`)': apiStates.apiPostFailed,
-        'The GET/LOAD fails (used for component testing only)': apiStates.apiGetFailed
-    }
+    default: apiStates.none,
+    states: [
+        apiStates.none,
+        apiStates.apiGetFailed,
+        apiStates.apiPostFailed
+    ]
+};
+
+const contactPreferencesGET200 = {
+    url: '/consumer/preferences',
+    method: httpVerbs.get,
+    responseStatus: httpStatusCodes.ok,
+    requestData: null,
+    responseData: getConsumerPreferences
+};
+
+const contactPreferencesPOST200 = {
+    url: '/consumer/preferences',
+    method: httpVerbs.post,
+    responseStatus: httpStatusCodes.ok,
+    requestData: null,
+    responseData: getConsumerPreferences
+};
+
+const contactPreferencesGET500 = {
+    url: '/consumer/preferences',
+    method: httpVerbs.get,
+    responseStatus: httpStatusCodes.internalServerError,
+    requestData: null,
+    responseData: {}
+};
+
+const contactPreferencesPOST500 = {
+    url: '/consumer/preferences',
+    method: httpVerbs.post,
+    responseStatus: httpStatusCodes.internalServerError,
+    requestData: null,
+    responseData: getConsumerPreferences
```

Comment:
Maybe use `{}` as the response data

```suggestion
    responseData: {}
```